lib/src/core/message/guild_emoji.dart

```diff
@@ -70,6 +117,12 @@ abstract class IGuildEmoji implements IBaseGuildEmoji {
   /// whether this emoji is animated
   bool get animated;
 
+  /// The user that created this emoji, not present if [fetchCreator] was not called
+  Cacheable<Snowflake, IUser>? get creator;
```

Comment:
This api feels weird, why not provide single function for featching creator without that kind of weird state?

Reply:
yeah sure. Then it would be better since caching cacheable in emojis is not neccessary